### PR TITLE
Changes the title of each webpage fixes # 43

### PR DESCRIPTION
**Pull request by RickyDoss**
Used the following commands in CLI for "React Helmet" plugin and "gatsby-transformer-json" npm installs:

**Command Line Interface Commands:**
npm install gatsby-plugin-react-helmet react-helmet
npm install gatsby-transformer-json

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -32,5 +32,7 @@ module.exports = {
     `gatsby-plugin-sharp`,
     `gatsby-transformer-remark`,
     `gatsby-plugin-sass`,
+    `gatsby-transformer-json`,
```
Comment (digitaldrk) [inline gatsby-config.js:35]: What is the purpose of `gatsby-transformer-json`?

File: src/pages/company_directory.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import Nav from "../components/nav"
 import Typography from "@material-ui/core/Typography"
 import TextField from "@material-ui/core/TextField"
 import CompanyCard from "../components/company_card"
+import { Helmet } from "react-helmet"
```
Comment (digitaldrk) [inline src/pages/company_directory.js:11]: Is it possible to add this to another Component that is imported in all pages? Then we would only need it once. Maybe `Header` is a good candidate 🤔

Comment (RickyDoss) [reply src/pages/company_directory.js:11]: I will look into how to do that

File: gatsby-config.js (hunk 1 above)
Comment (RickyDoss) [reply gatsby-config.js:35]: It was recommended for adding in a more variable title but I may be able to remove that and still have the React Helmet work

File: src/pages/company_directory.js (hunk 2 above)
Comment (RickyDoss) [reply src/pages/company_directory.js:11]: oh nice, yes header did work. I'll have to pick your brain on that sometime for my own knowledge

File: gatsby-config.js (hunk 1 above)
Comment (RickyDoss) [reply gatsby-config.js:35]: Removed the JSON plugin and the React Helmet does still work without it so I will keep that out for now.

**Comment by RickyDoss** [comment]
Should I push new changes to this PR or scrap this and make a new one?

**Pull request merged**